dsc/config.py

```diff
@@ -37,6 +38,10 @@ def item_table_name(self) -> str:
             raise OSError("Env var 'ITEM_TABLE_NAME' must be defined")
         return value
 
+    @property
+    def retry_threshold(self) -> int:
+        return int(os.getenv("RETRY_THRESHOLD", "20"))
+
```

Comment:
Can be simplified to pass an `int` to `os.getenv`:

```python
os.getenv("RETRY_THRESHOLD", 20)
```

Reply:
@jonavellecuerdo - not 100% sure about that?  If we want to be gauranteed an integer coming out of this, my working assumption has always been that env var values always start as strings.

I think wrapping with `int()` and defaulting to a string `"20"` is a pretty standard way of doing it?

Reply:
I retract my request!